lang/en/tool_pdfpages.php

```diff
@@ -31,10 +31,10 @@
 $string['error:wkhtmltopdfpath'] = 'Cannot establish path to wkhtmltopdf, please check setting tool_pdfpages|wkhtmltopdfpath.';
 $string['error:urltopdf'] = 'URL to PDF conversion could not be completed.';
 $string['settings:accesskeyheading'] = 'Access key settings';
-$string['settings:accesskeyttl'] = 'TTL (Time To Live)';
+$string['settings:accesskeyttl'] = 'TTL (Time to live)';
 $string['settings:accesskeyttl_desc'] = 'The time in seconds for access keys to live before expiring';
 $string['settings:convertersheading'] = 'Converter settings';
-$string['settings:chromiumpath'] = 'chromium binary';
-$string['settings:chromiumpath_desc'] = 'The path to the chrome/chromium binary';
-$string['settings:wkhtmltopdfpath'] = 'wkhtmltopdf binary';
-$string['settings:wkhtmltopdfpath_desc'] = 'The path to wkhtmltopdf binary';
+$string['settings:chromiumpath'] = 'Path to Chromium';
```

Comment:
I can accept the `Time to live` change, as this is an initialisation, but 'Chromium' is a noun (proper name), this shouldn't be changed, (see https://docs.moodle.org/dev/Coding_style#Strings)

Reply:
Sorry, I am confused cause it sounds like you are agreeing with the change (that it becomes a capital).
```
Capitals should only be used when:

    starting a sentence, or
    starting a proper name, like Moodle
```
As you said 'Chromium' is a proper name so should start in capitals.

Reply:
Oh my bad lol, I misread the commit, :smile_cat: you are correct. I'm not sure about the noun for wkhtmltopdf though, I think it is supposed to be wkHTMLtoPDF, but I can't find anything definitive.

Reply:
I googled `wkhtmltopdf` and found https://wkhtmltopdf.org/ (obviously) which just keeps it all on lowercase so I say we just leave it as is.